public/index.html

```diff
@@ -24,6 +24,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
     <title>React App</title>
   </head>
   <body>
```

Comment:
- [ ] Good work on implementing the design, however kindly remove the white space around the app so the core design can populate the whole page

|Your design|How it should look|
|-|-|
|![Screenshot_20221014_233351](https://user-images.githubusercontent.com/25227022/195953536-bd2e4cad-f05e-48ae-b95a-648696c56427.png)|![Screenshot_20221014_233652](https://user-images.githubusercontent.com/25227022/195953596-d43201a6-4828-4cdd-a5c9-586e4193c11d.png)|

Reply:
@Ismailco thanks alot for mentioning it i already removed it